src/sinks/gcp/cloud_storage.rs

```diff
@@ -62,6 +63,7 @@ pub struct GcsSinkConfig {
     filename_append_uuid: Option<bool>,
     filename_extension: Option<String>,
     encoding: EncodingConfig<Encoding>,
+    #[serde(default = "Compression::config_default")]
     compression: Compression,
```

Comment:
Could this use the generic `util::Compression` here with an alternate default? I see that it uses a different case convention, but that could be handled in `util` with serde alias attributes.

Reply:
I just removed internal `Compression` and proceed with `uitl::Compression`. Only difference is `snake_case` vs `lowercase`, but for `None` / `Gzip` no difference between this rename attributes.

Reply:
Ah, yes, of course. I was thinking of one of the other GCS attributes where we do encode it when building the request, so a specific rename convention is required.